modules/deployment/src/test/resources/classloading-troubleshooting/errors/app-jms-properties-resource-not-found

```diff
@@ -22,4 +22,18 @@ Couldn't read from file jms.properties: Couldn't find resource: jms.properties n
                           ------------------------
  
  Current classloader in context is (App Class Loader) 
- The resource exists in the following context (Domain Class Loader) but is not exported. Try exporting the resource by modifying the mule-artifact.json file
\ No newline at end of file
+ The resource exists in the following context (Domain Class Loader) but is not exported. Try exporting the resource by modifying the mule-artifact.json file
```

Comment:
arent't these files used when running without modules as well?

Reply:
This is what I mentioned that I had to fix (I already had)